week-2/extended-project/valeria_and_friends/classes/system.rb

```diff
@@ -0,0 +1,105 @@
+require "csv"
+
+class System
+  TEXT_INSTRUCTIONS = [
+    "+------------------------------------------+",
+    "  5 Star Wars curiosities you didn't know:",
+    "+------------------------------------------+",
+    "  1. Most expensive vehicles",
+    "  2. Fastest vehicles",
+    "  3. Tallest people",
+    "  4. Heaviest people",
```

Comment:
Can you tell me how this information about overweight people could be significant to us?

Reply:
It was just a fact, we didn't really had any other in mind. You're right. We'll probably change it if we release another version of this :) thanks for noticing it.